channels/rdpsnd/server/rdpsnd_main.c

```diff
@@ -591,24 +595,18 @@ void rdpsnd_server_context_reset(RdpsndServerContext *context)
 
 void rdpsnd_server_context_free(RdpsndServerContext* context)
 {
-	if (!context->priv->StopEvent)
```

Comment:
Any reason why you removed that safety check: code trying to free a running channel ?
I don't know any code that would sanely do that but the check what there to ensure it !